spyder_unittest/widgets/configdialog.py

```diff
@@ -101,6 +116,7 @@ def __init__(self, frameworks, config, parent=None):
 
         self.ok_button = self.buttons.button(QDialogButtonBox.Ok)
         self.ok_button.setEnabled(False)
+        self.coverage_checkbox.setEnabled(False)
```

Comment:
I followed the pattern used for the ok_button code. 

~I first set it to disabled. Then if pytest is chosen, it gets enabled. If unittest or nose is chosen, it gets unchecked AND disabled. If I comment this out, then the checkbox is initially enabled even before a framework is chosen. I guess it's fairly moot though, since if you check it and then choose nose or unittest, then the checkbox will get unchecked and disabled.~

I think I get your question now. OK I moved it up to where I create the checkbox :smile: